### PR TITLE
Build debian package in SONiC build environment

### PR DESCRIPTION
**Pull request by renukamanavalan**
Successfully built debian package

```
remanava@remanava-dev-vm:~/sonic/fork/test-integrate/target/debs/bullseye$ ls -lrt | tail -n 2
-rw-r--r-- 1 remanava remanava   2686680 Jun 15 01:27 sonic-device-health_1.0.0-0_amd64.deb
-rw-r--r-- 1 remanava remanava   5628019 Jun 15 01:27 sonic-device-health_1.0.0-0_amd64.deb.log
remanava@remanava-dev-vm:~/sonic/fork/test-integrate/target/debs/bullseye$ 
```

### CODE REVIEW

File: lom/Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ UID = $(shell id -u)
 GUID = $(shell id -g)
 
 
-GO := /usr/local/go/bin/go
+GO := /usr/local/go1.20.3/go/bin/go
```
Comment (GouthamBanala) [inline lom/Makefile:10]: This is creating error on my build  container. I have to switch to /usr/local/go/bin/go

File: lom/Makefile [hunk 2]
```diff
@@ -68,7 +72,12 @@ endef
 
 .PHONY: all clean format_code
 
-all: format_code $(CMN_C_LIB) $(ENGINE_BIN) $(LIB_TEST) $(ENGINE_TEST) $(PLUGIN_MANAGER_TEST) \
+origall: format_code $(CMN_C_LIB) $(ENGINE_BIN) $(LIB_TEST) $(ENGINE_TEST) $(PLUGIN_MANAGER_TEST) \
+	$(SONIC_DBCLIENT_TEST) $(SONIC_LINK_CRC_TEST) $(PLUGINS_COMMON_TEST) $(PY_TEST) \
+	$(PLUGIN_MANAGER_BIN)
+
+
+all: $(CMN_C_LIB) $(ENGINE_BIN) $(LIB_TEST) $(ENGINE_TEST) $(PLUGIN_MANAGER_TEST) \
```
Comment (GouthamBanala) [inline lom/Makefile]: why format code is removed ?

File: lom/src/lib/lib_test/config/procs.conf.json [hunk 3]
```diff
@@ -1,42 +1,44 @@
...
-	"proc_4": {
-		"GenericPluginDetection": {
-			"name": "GenericPluginDetection",
-			"version": "1.0",
-			"path": ""
-		}
-	}
-}
\ No newline at end of file
+    "procs": {
```
Comment (GouthamBanala) [inline lom/src/lib/lib_test/config/procs.conf.json:2]: change ...we can discuss .

File: lom/Makefile (hunk 1 above)
Comment (renukamanavalan) [reply lom/Makefile:10]: Rebuild slave container

File: lom/Makefile (hunk 2 above)
Comment (renukamanavalan) [reply lom/Makefile]: Added back

**Review by prithviraghavtm — APPROVED** [review]

**Review by GouthamBanala — APPROVED** [review]

**Pull request merged**